packages/plugin-e2e/src/fixtures/commands/readProvisionedDataSource.ts

```diff
@@ -0,0 +1,26 @@
+import { TestFixture } from '@playwright/test';
+import { promises } from 'fs';
+import path from 'path';
+import { parse as parseYml } from 'yaml';
+import { PluginFixture, PluginOptions } from '../../api';
+import { DataSourceSettings, ReadProvisionedDataSourceArgs } from '../../types';
+import { PlaywrightCombinedArgs } from '../types';
+
+type ReadProvisionedDataSourceFixture = TestFixture<
+  <T = any>(args: ReadProvisionedDataSourceArgs) => Promise<T>,
+  PluginFixture & PluginOptions & PlaywrightCombinedArgs
+>;
+
+const readProvisionedDataSource: ReadProvisionedDataSourceFixture = async ({ provisioningRootDir }, use) => {
+  await use(async ({ fileName: filePath, name }) => {
+    const resolvedPath = path.resolve(path.join(provisioningRootDir, path.join('datasources', filePath)));
```

Comment:
**Nit:** I guess we could just do the following?
```suggestion
    const resolvedPath = path.resolve(path.join(provisioningRootDir, 'datasources', filePath));
```

Reply:
Yep, good shout! Thanks